Cargo.toml

```diff
@@ -13,3 +13,13 @@ opt-level = 3
 # Uncomment to profile on Linux:
 # debug = true
 # lto = false
+
+[replace]
+# If you need to temporarily test Servo with a local fork of some upstream
+# crate, add that here. Use the form:
+#
+#     "<crate>:<version>" = { path = "/path/to/local/checkout" }
+#
+# For example:
+#
+#     "bindgen:0.23.1" = { path = "/home/fitzgen/rust-bindgen" }
```

Comment:
It's just the replacement that I actually had ;)

I can remove this example, if you don't think it is helpful beyond the `"<crate>:<version>" = ...` part above.